packages/web-components/examples/codesandbox/components/table-of-contents/index.html

```diff
@@ -25,7 +25,22 @@
       padding-top: 1rem;
       padding-bottom: 2rem;
     }
+
+    caem-accordion {
+      display: block;
+      padding: 2rem 0;
+    }
+
+    dds-carousel {
+      --dds--carousel--overflow-hint-size: 0;
+      padding: 2rem 0;
+    }
   </style>
+  <!-- CDN artifact -->
+  <script
+    type="module"
+    src="https://1.www.s81c.com/common/carbon/carbon-for-aem/tag/canary/accordion.min.js"
```

Comment:
```suggestion
    src="https://1.www.s81c.com/common/carbon/carbon-for-aem/tag/latest/accordion.min.js"
```

Minor/nitpicking - Let's use the `latest` tag for the `<caem-accordion>` component.